### DISCUSSION TITLE
Will UI be ever optimized for mobile/tablet/touchscreen device.

### DISCUSSION THREAD

**Discussion by shiroanon** [Comfy-Org/ComfyUI_frontend#1741] [General]
I don't know about the masses but i prefer running comfy on my android tablet , but somehow its worse every time i use it , weird disconnects , glitch , not being able to use almost every good features like group nodes etc .

**Comment by LukeG89** [comment]
There are plans to improve touch/mobile support:
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/2889

Btw, there is this custom nodes to improve touchscreen support: [ComfyUI-Touch-Gestures](https://github.com/Iemand005/ComfyUI-Touch-Gestures)
